client/src/Poem.jsx

```diff
@@ -15,6 +16,7 @@ const Poem = ({ poem }) => {
     const NBSP = '\u00A0'; // Unicode value for non-breaking space
     return <PoemLine line={(line.trim().length === 0) ? NBSP : line} key={index} />;
   });
+  const poemCommentsURL = '/api/comments/'.concat(poem.id);
```

Comment:
Mozilla suggests [using `+`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/concat#Performance) over `concat`. Also, I think it would be fine to inline this operation in the `CommentBox` element below.